src/app/_helpers/authentication/auth.guard.ts

```diff
@@ -11,9 +11,7 @@ export class AuthGuard implements CanActivate {
               private router: Router) {
   }
 
-  canActivate(
-    next: ActivatedRouteSnapshot,
-    state: RouterStateSnapshot): boolean {
+  canActivate(): boolean {
```

Comment:
@HonzaDudek only have changes in the pull request that are related to the feature